### DISCUSSION TITLE
pnpm saas infra bootstrap

### DISCUSSION THREAD

**Discussion by norfolkpine** [apptension/saas-boilerplate#616] [Q&A]
I followed the guide to **deploy to AWS**, got error when running `pnpm saas infra bootstrap`

The error says
`----------
"staging" is set as a current environment stage. Live AWS session credentials are being used.
----------

 ›   Error: Failed to load environmental variables from SSM Parameter Store using chamber: Error: Command failed: chamber export staging --format dotenv
 ›   Error: Failed to list store contents for service staging: operation error SSM: GetParametersByPath, https response error StatusCode: 400, RequestID: 16e35128-d0cf-4925-aea4-60c903816e06,
 ›    api error UnrecognizedClientException: The security token included in the request is invalid
 ›`
 
 
---

In step 3 it says **Before deploying the infrastructure to AWS, you must set all required environmental variables!**, got example of how to do this in shell/terminal? (stuck at step 3)

Thanks in advance

**Comment by mkleszcz** [comment]
Looks like your AWS configuration is invalid or `aws-vault` session expired. Check the [Configure AWS credentials](https://docs.demo.saas.apptoku.com/aws/deploy-to-aws/configure-aws-credentials) documentation
